### PR TITLE
Clarify that SR-IOV Network Operator changes are optional

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1822080

This may cherry-pick to 4.3 as well, but may conflict.

### CODE REVIEW

**Comment by zshi-redhat** [comment]
In the generated preview, it we scroll down to the bottom of  [`Installing the SR-IOV Network Operator`](https://bz-1822080--ocpdocs.netlify.app/openshift-enterprise/latest/networking/hardware_networks/installing-sriov-operator.html): it doesn't have `Optional` prefix in the hyper link to `Configuring the SR-IOV Network Operator`

File: modules/nw-sriov-configuring-operator.adoc [hunk 1]
```diff
@@ -5,9 +5,20 @@
...
+====
+
 The SR-IOV Network Operator adds the `SriovOperatorConfig.sriovnetwork.openshift.io` Custom Resource Definition (CRD) to {product-title}.
 You can configure the SR-IOV software compoenents by modifying the default 
-SriovOperatorConfig Custom Resource (CR).
+SriovOperatorConfig Custom Resource (CR). The Operator supports the following optional behavioral changes:
+
+- Disabling the Network Resources Injector
+- Disabling the Operator admission web controller
+- Configuring a custom node selector
```
Comment (pliurh) [inline modules/nw-sriov-configuring-operator.adoc:21]: To make it more clear, we could say.
Configuring a custom node selector for the sriov-network-config-daemon. It runs on all the worker nodes by default.

**Review by bergerhoffer — COMMENTED** [review]
LGTM! Though we aren't supposed to be hard wrapping lines anymore, but I don't think we've been holding up PRs over it.

**Comment by jboxman** [comment]
@bergerhoffer thanks!

**Pull request merged**